platform/engineering/official_github_teams_process

```diff
@@ -0,0 +1,38 @@
+# GitHub Teams on for VA.gov
+
+## What is GitHub Teams, and why are we talking about it?
+
+GitHub Teams give organizations the ability to create groups of members and control access to repositories. Team members can be granted read, write, or admin permissions to specific repositories.
+For more information on GitHub teams, please see [GitHub docs](https://help.github.com/en/enterprise/2.19/admin/user-management/creating-teams).
+
+At the moment, some teams have their own GitHub Teams, while others do not. GitHub Teams’ benefits and functions are not fully understood across VA.gov. Our goal is to provide a basic understanding of GitHub teams, establishing a logical structure of teams first. Then we can improve the process of granting access and developers’ experience on VA.gov. 
+
+In the future, we will be using *Official GitHub Teams* as a source of truth for team rosters, access, and auditing purposes for VSP. The process of creating and maintaining Official GitHub Teams is below. 
+
+## GitHub Team Process
+
+### Creating a GitHub Team 
+
+* Work with your DEPO, Tech, and Team Lead to change an existing team to meet the requirements below or [create](https://help.github.com/en/enterprise/2.19/admin/user-management/creating-teams)  an *Official GitHub Team* on VA.gov. 
+    * All *Official GitHub Team* names should start with `vagov-` followed by the team name, which indicates the type of team this is. Examples of this are `vfs-*` (Veteran-facing Services) for teams working on an application or building tools and services on VA.gov.  Another is `vsp-*` (Veteran-facing Services Platform) for teams working on the platform supporting VFS teams. Examples would be `vagov-vsp-operations or `vagov-vfs-public-website`. If those two team types do not fit your team, work with the Platform team to create a logical team type prefix. 
```

Comment:
Not to keep beating on the same thing, but I hate the "vagov-" prefix.

Reply:
I kind of feel like this should be a small decision tree.

Is your team within the VSP program? Your prefix is vsp-
Is your team within the Lighthouse program? Your prefix is <?>-
Is your team within the VSA program? Your prefix is vsa-
Do you fit none of these? Your prefix is vfs-

Reply:
I hear you on this. The problem when we end up having 50 teams in place, it is going to be a memory game to try and remember all the different titles teams have. cms-, dvp-, vsp-, vsa-, vfs-, ect. I am trying to make it simple for not only us, but anyone to filter by teams, so if they want a global veiw on va.gov then we use vagov-, if they want just the vsa teams they can use vsa, but it will come up as vagov-vsa-*. I'm open to other naming, I am just trying to make our lives a bit easier in the future.

Reply:
VFS better name for the prefix